vyper/old_codegen/global_context.py

```diff
@@ -224,6 +224,7 @@ def add_globals_and_events(self, item):
     def parse_type(self, ast_node, location):
         return parse_type(ast_node, location, sigs=self._contracts, custom_structs=self._structs,)
 
+    # TODO this is dead code
     def get_nonrentrant_counter(self, key):
```

Comment:
Can we remove this dead code?

Reply:
I don't want to now just in case it's a larger project

Reply:
just remove it and push, if it causes CI failures then take it out.

Reply:
> just remove it and push, if it causes CI failures then take it out.

done, see latest commit